api/rest/v2/sparql.md

```diff
@@ -0,0 +1,182 @@
+content on the web. It leverages the graph structure innate in the data it 
+queries to return specific information.
+
+**Note:** Data Commons only supports a limited subset of SPARQL functionality at this time: specifically, only the keywords `WHERE`, `ORDER BY`, `DISTINCT`, and `LIMIT` are supported.
+
+## Request
+
+<div class="api-tab">
+  <button id="post-button" class="api-tablink">POST request</button>
+</div>
+
+<div id="POST-request" class="api-tabcontent api-signature">
+URL: https://api.datacommons.org/v2/sparql
+
+Header: X-API-Key: <var>API_KEY</var>
+
+JSON data: { "sparql": "<var>SPARQL_QUERY</var>" }
+</div>
+
+<script src="/assets/js/syntax_highlighting.js"></script>
+<script src="/assets/js/api-doc-tabs.js"></script>
+
+### Query parameters
+
+| Name                                                | Type   | Description                                                                                                                                                     |
+| --------------------------------------------------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| key <br /> <required-tag>Required</required-tag>    | string | Your API key. See the the section on [authentication](getting_started.md#authentication) for instructions on how to get a key. |
+| sparql <br /> <required-tag>Required</required-tag> | string | A SPARQL query string.<br/>In the query, each variable should have a `typeOf` condition, for example, <code>?<var>VARIABLE_NAME</var> typeOf City</code>.                                              |
+{: .doc-table }
+
+## Response
+
+The response looks like:
+
+<pre>
+{
+  "header": [
+    <var>STRING</var>
+  ],
+  "rows": [
+    {
+      "cells": [
+        {
+          "value": <var>STRING</var>
+        }
+      ]
+    },
+    ...
+  ]
+}
+</pre>
+{: .response-signature .scroll}
+
+### Response fields
+
+| Name   | Type   | Description                                                                      |
+| ------ | ------ | -------------------------------------------------------------------------------- |
+| header | list   | List of strings corresponding to the query variables.                            |
+| rows   | list   | List of `row` objects, with each containing a list of cells and its cell values. |
+| cells  | object | Contains string field `value` corresponding to the queried variable.             |
+{: .doc-table}
+
+## Examples
+
+### Example 1: Query the Data Commons knowledge graph with SPARQL
+
+Retrieve a list of 10 biological specimens (DCID: `BiologicalSpecimen`) in
+reverse alphabetical order.
+
+Request:
+{: .example-box-title}
+
+<pre>
+curl --request POST \
+  --url https://api.datacommons.org/v2/sparql \
+  --header 'X-API-Key: <var>API_KEY</var>' \
+  --data '{
+            "sparql": "SELECT ?name \
```

Comment:
here should be "query" not "sparql"?

Reply:
Right, good catch, thank you.

Reply:
I realized that the sparql v2 change is not pushed to prod. Maybe run through the examples here when that's done by next Monday.

Reply:
Actually, I just realized that the example I ran before did use "sparql" and not "query". Are you sure that's correct? Maybe both are supported?